### DISCUSSION TITLE
Welcome to NeutronHeaven Discussions!

### DISCUSSION THREAD

**Discussion by secretmeister** [secretmeister/NeutronHeaven#4] [Announcements]
This is an easy way to have conversations with other players, as well as me, the pack developer! Discussions can be converted into proper issues, and other things. Feel free to start any with questions, I am haunting this place looking for feedback!

**Comment by Talljack459** [comment]
Hey! I'm officially completely lost. I've hit a point where I can't continue without wood. But I cant seem to figure out how to get slime grass. Help!
